### DISCUSSION TITLE
Entity data history

### DISCUSSION THREAD

**Discussion by jerome6994** [home-assistant/frontend#18056] [Configuration]
Good morning

For the practical sake of optimizing the data and the size of the database, I was wondering if it was possible by default to record nothing?
But by adding a check box in front of each entity the user could choose to historicize the entity data.
This would therefore feed the recorder and in this way would only really record the bare minimum.

The ideal would be for this box to be available on the device screens, of the complete list of entities for mass processing if necessary in advanced mode.

thanks in advance

Jérôme

**Comment by ildar170975** [comment]
1. You know, some people prefer using yaml. Defining recorder's settings in UI may not make them happier...
2. "by default to record nothing" - this would be a really breaking change for many users.

**Comment by jerome6994** [comment]
Good evening

Indeed, these are changes but not insurmountable.
You are making important changes to the UI and this is part of it.
This helps make the tool even more accessible.
It's not about doing it overnight but gradually.
Less Yaml = more users

In France many come from Jeedom which is on this principle which protects the least alert people with yaml configurations

**Reply by ildar170975** [reply]
> Less Yaml = more users

and probably less possibilities. But this question is far beyond this particular discussion.
